blobfuse-nightly.yml

```diff
@@ -0,0 +1,235 @@
+# BlobFuse Nightly Build-Sanity Pipeline
+
+# Run this Pipeline whenever any change happens in selected branch
+#trigger:
```

Comment:
I was thinking of running this on every commit along with running it on scheduled time so that we can catch the failures early as soon as commit is pushed. But later found another pipeline is already running so left it assuming we may be able to merge both of them.

Reply:
I vote for removing it as merging will cause confusion.

Reply:
Sure will remove the trigger part and keep only the schedule part.